modules/ROOT/pages/rest-sdk/configure-operations.adoc

```diff
@@ -3,22 +3,35 @@ ifndef::env-site,env-github[]
 include::../_attributes.adoc[]
 endif::[]
 
-== Operations Identifier
-REST SDK heavily relies on the operation identifier for the resources’ classes generation, that operation identifier could be fetched from the OAS model under `operationId` or if it’s from a RAML model it will default to null.
+== Operation Identifiers
 
-The importance of the identifier is that it must not collide with other resources (nor triggers, or any other type of REST SDK component) and that it can be modified in the descriptor by comprehension. REST SDK does provide a default mechanism for generating that resource identifier, in which if there’s a valid operationId (from an OAS context) with a non null value, it will use it, or it execute the fallback of using the HTTP method plus the resource name (which is unique in an API).
+Use an operation identifier in REST SDK to generate classes for resources.
+Fetch an operation identifier from either an OAS model
+under `operationId` or a RAML model. If fetched from a RAML model, the operation
```

Comment:
this is not correct, specifically the `or a RAML model.`, the operationId will be null when coming from a RAML.